app/dashboard/utils.py

```diff
@@ -298,8 +298,13 @@ def get_web3(network, sockets=False):
         web3.main.Web3: A web3 instance for the provided network.
 
     """
-    if network in ['mainnet', 'rinkeby', 'ropsten']:
-        if sockets:
+    if network in ['mainnet', 'rinkeby', 'ropsten', 'testnet']:
+        if network == 'mainnet' and is_polygon:
+            network = 'polygon-mainnet'
+        elif network == 'testnet':
```

Comment:
Should this be `elif network == 'testnet' and is_polygon:` or is `testnet` only used for polygon anyway so it is redundant?

Reply:
yes it's only used for polygon

Reply:
yes it's only used for polygon